Dhruvkhanna/code cleanup

dhruvkhanna38 opened the pull request:
Merging branches
- https://github.com/make4all/psst/tree/noramorsi/keyboard 
- https://github.com/make4all/psst/tree/origin/jen-architecting 
to main

Commit:
Merge branch 'noramorsi/handlers'

Commit:
Buggy alternative way to do slope

Commit:
Added a slope demo that plays a tone representing the current slope

Commit:
jen's in-progress code

venkateshpotluri committed:
partial bug fix.

Commit:
commitment for venkatesh

Commit:
Merge branch 'origin/jen-architecting' of https://github.com/make4all/sonification into jen/testing

noramorsi committed:
basic set up of keyboard page; onclick not triggering

noramorsi committed:
set up space in page for focus

noramorsi committed:
mapping created

noramorsi committed:
tones playing, need to fix auto-play

noramorsi committed:
attempt at fixing tones continuing

noramorsi committed:
basic keyboard visual set up, no change visually

noramorsi committed:
note ends when key up

noramorsi committed:
style changes

noramorsi committed:
black keys added, do not wrap like white keys

noramorsi committed:
shifted down an octave

noramorsi committed:
single octave with black keys included

noramorsi committed:
cleaning up constants

noramorsi committed:
mouse position sonification demo added

noramorsi committed:
expanded to include 3rd octave

venkateshpotluri committed:
buggy running average code.

venkateshpotluri committed:
debugging logs.

venkateshpotluri committed:
almost done piano.

dhruvkhanna38 committed:
Created a new path /DemoCopyStream with button to copy data collected from microbit as .csv

dhruvkhanna38 committed:
feature: CopyToClipboardHandler add to dashboard.tsx

dhruvkhanna38 committed:
Merge branch 'noramorsi/keyboard' into dhruvkhanna/code-cleanup

dhruvkhanna38 committed:
Merge branch 'origin/jen-architecting' into dhruvkhanna/code-cleanup

dhruvkhanna38 committed:
fix: build

dhruvkhanna38 committed:
Merge branch 'dhruvkhanna/demo-copy-stream' into dhruvkhanna/code-cleanup

dhruvkhanna38 committed:
merging noramorsi/keyboard and jen-architecting to main

dhruvkhanna38 committed:
fix: Dashboard.tsx Cleanup

dhruvkhanna38 committed:
fix: Deleting Copy Handler

venkateshpotluri commented:
Please perform the following tests before we merge. Possible include these as test cases that could run as a part of the CI/CD pipeline.
- [ ] check the keyboard demo on the dashboard.
- [ ] in a separate code file, try chaining handelers and test if those work. e.g. a filter range handeler with a sound effect.
- [ ] test handeler-output combinations using the dashboard to ensure nothing breaks.

dhruvkhanna38 commented:
@venkateshpotluri the tests above are working for this pull request. It is good to merge.

I removed some deprecated code and updated the comments for some of the classes.

dhruvkhanna38 committed:
Updating comments and removing deprecated code
